packages/sources/src/components/viewer-grid/viewer-grid.models.ts

```diff
@@ -30,6 +31,7 @@ export interface ViewerGrid<TemplateFnReturnType> {
   overlay: TemplateFnReturnType;
   overlayOpen: boolean;
   noOverlay: boolean;
+  initialMainSize: "small" | "medium" | "large";
```

Comment:
Het zetten van de initial main size is optioneel, dat moeten we terugzien in het model van het component.

Deze property moet dus optioneel worden.

Reply:
Aangepast.